### ISSUE TITLE
Where is the result video?I have taken the last setp, but the video is not found.

### ISSUE THREAD

**Issue by GreatWildFire** [yerfor/GeneFace#214]
warmup_updates: 0, weight_decay: 0, with_att: True, work_dir: checkpoints/May/lm3d_radnerf_torso, 
10/12 01:45:07 PM GPU available: True, GPU used: [0]
trainval: Smooth head trajectory (rotation and translation) with a window size of 7
Saved 16khz wav file to data/raw/val_wavs/zozo_16k.wav.
Loaded pre-extracted 3D landmark sequence from infer_out/May/pred_lm3d/zozo.npy!
| Unknow hparams:  []
val: Smooth head trajectory (rotation and translation) with a window size of 7
Setting up [LPIPS] perceptual loss: trunk [alex], v[0.1], spatial [off]
Loading model from: /home/admin1/anaconda3/envs/geneface/lib/python3.9/site-packages/lpips/weights/v0.1/alex.pth
| load 'model' from 'checkpoints/May/lm3d_radnerf/model_ckpt_steps_250000.ckpt'.
Loaded Head Model from checkpoints/May/lm3d_radnerf
Loaded state_dict of Head Model to the RADNeRFTorso Model
| load 'model' from 'checkpoints/May/lm3d_radnerf_torso/model_ckpt_steps_250000.ckpt'.
The tmp imge dir is infer_out/May/pred_video/tmp_imgs/zozo_radnerf_torso_smo.
NeRF is rendering frames...: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 272/272 [00:20<00:00, 13.30it/s]
The synthesized video is saved at infer_out/May/pred_video/zozo_radnerf_torso_smo.mp4

 I have installed the ffmpeg and change libopenh26.so.6 to libopenh264.so.5.But the video is not found all the time. I need help plz

~$ ffmpeg -version
ffmpeg version 5.1 Copyright (c) 2000-2022 the FFmpeg developers
built with gcc 9 (Ubuntu 9.4.0-1ubuntu1~20.04.2)
configuration: --prefix=/usr/local/ffmpeg
libavutil      57. 28.100 / 57. 28.100
libavcodec     59. 37.100 / 59. 37.100
libavformat    59. 27.100 / 59. 27.100
libavdevice    59.  7.100 / 59.  7.100
libavfilter     8. 44.100 /  8. 44.100
libswscale      6.  7.100 /  6.  7.100
libswresample   4.  7.100 /  4.  7.100

**Comment by huozhong-in** [comment]
same issue.

**Comment by cillyfly** [comment]
It may be that ffmpeg lacks h264 encoding. You can try to replace mpeg4 or the encoder on your machine.

1. check the encode in you device
 run `ffmpeg -encoders` in console
2. pick video encode like 'mpeg4'
<img width="832" alt="image" src="https://github.com/yerfor/GeneFace/assets/6388202/e721ab43-44da-471b-82a0-63fe26b3d30f">

4. change the code 
<img width="1346" alt="image" src="https://github.com/yerfor/GeneFace/assets/6388202/6b27a541-1614-4286-9a44-d45de8a7284e">
AND run again. you will get the video
